deploy-as-code/helm/charts/municipal-services/ws-services/values.yaml

```diff
@@ -115,6 +115,8 @@ env: |
     value: {{ index .Values "java-args" | quote }}
   - name: SERVER_PORT
     value: "8080"
+  - name: JAVA_ENABLE_DEBUG
+    value: "true"
```

Comment:
Do u want to enable debug mode for all envs? If not, make sure u override these value through the respective env.yaml.

Reply:
want to enable it for all env